starter-code/css/style.css

```diff
@@ -1,77 +1,40 @@
-input {
-  border: solid 1px black;
-}
-
-.container {
-
-}
-
-.info-container {
-
-}
-
-.item {
-
+.header-content {
```

Comment:
Great Job.  Looks great and works very well.  I noticed the issues pending list you left at the top, also good to realize what is missing even when you have a very good-looking site.